app/logic/fileHandler.py

```diff
@@ -12,41 +12,54 @@ def __init__(self,files=None, courseId=None, eventId=None):
         if courseId:
             self.path = os.path.join(self.path, app.config['files']['course_attachment_path'], str(courseId))
         elif eventId:
-            self.path = os.path.join(self.path, app.config['files']['event_attachment_path'], str(eventId))
+            self.path = os.path.join(self.path, app.config['files']['event_attachment_path'])       #Removed event ID from how the file path is being saved.
```

Comment:
Instead of the comment at the end of the line it should be above the line. It makes the line shorter, and you will read it before the line itself. I would also re-word it to say what is happening or the reason, rather than what changed. Like, "Event ID is included in the filename, not in the path"